test/e2e/framework/util.go

```diff
@@ -128,8 +128,29 @@ const (
 
 	// How long claims have to become dynamically provisioned
 	ClaimProvisionTimeout = 5 * time.Minute
+
+	// When these values are updated, also update cmd/kubelet/app/options/options.go
+	defaultPodInfraContainerImageName    = "gcr.io/google_containers/pause"
+	defaultPodInfraContainerImageVersion = "3.0"
```

Comment:
Ah, I just copied them from options.go where default was a better name.
Here, obviously, it's not a default but the current value